reference/configuration/framework.rst

```diff
@@ -1560,6 +1561,16 @@ paths
 This option allows to define an array of paths where the component will look
 for translation files.
 
+.. _reference-translator-default_path:
+
+default_path
+.....
+
+**type**: ``string`` **default**: ``[]``
```

Comment:
the default value looks wrong to me (this should be `%kernel.project_dir%/translations`)

Reply:
Thanks, you are right